seonshine_frontend/src/apis/hooks/authApi.hook.ts

```diff
@@ -1,25 +1,26 @@
+import { useNavigate } from 'react-router-dom';
 import { toast } from 'react-toastify';
 
 import { useMutation, UseMutationResult } from '@tanstack/react-query';
 
 import { LoginSchemaType } from '@/pages/login/schemas';
 
+import { errorMessages } from '@/constants/errorMessages';
+
 import { login, LoginResponseType } from '../auth';
-import { useAuth } from '@/hooks/useAuth';
 
 export const useLoginApi = (): UseMutationResult<LoginResponseType, unknown, LoginSchemaType> => {
-  const {login: handleSuccessLogin } = useAuth();
+  const navigate = useNavigate();
 
   return useMutation({
     mutationFn: async (payload) => {
       return login(payload.employeeId, payload.password);
     },
-    onSuccess: (data) => {
-      toast.success('Login successfully!');
-      handleSuccessLogin(data, data.token, false);
+    onSuccess: () => {
```

Comment:
this file just export function. onSuccess and onError must be handle from login component